core/translate/expr.rs

```diff
@@ -2074,3 +2074,9 @@ pub fn get_name(
         _ => fallback(),
     }
 }
+
+// Sanitaizes a string literal by removing single quote at front and back
+// and escaping double single quotes
```

Comment:
Can this be a doc comment instead of?

```suggestion
/// Sanitaizes a string literal by removing single quote at front and back
/// and escaping double single quotes
```

Reply:
done